lms/djangoapps/support/static/support/jsx/entitlements/components/Table/Table.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+import { Button, Hyperlink, Table } from '@edx/paragon';
+
+const entitlementColumns = [
+  {
+    label: 'User',
+    key: 'user',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Course uuid',
+    key: 'course_uuid',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Enrollment',
+    key: 'enrollment_course_run',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Mode',
+    key: 'mode',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Expired At',
+    key: 'expired_at',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Created',
+    key: 'created',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Modified',
+    key: 'modified',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Order',
+    key: 'order_number',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Actions',
+    key: 'button',
+    columnSortable: false,
+    hideHeader: false,
+    onSort: () => {},
+  },
+];
+
+const EntitlementTable = (props) => {
+  const entitlementData = props.entitlements.map((entitlement) => 
+    Object.assign({}, entitlement, {
+      expired_at: entitlement.expired_at ? moment(entitlement.expired_at).format('MMM. D, YYYY') : '',
+      created: moment(entitlement.created).format('MMM. D, YYYY'),
```

Comment:
If it's `expiredAt` should these be `createdAt` and `modifiedAt`?

Reply:
This are basically direct lifts of the names in the model, expiredAt was used instead of expired to more clearly be a date/time since expired seems like it would be a boolean. Whereas created and modified I feel like are more expected to be dates anyway. Think it would still be clearer to put the 'At' on the end of those?

Reply:
`created` and `modified` still seem more `boolean`y than `createdAt` or `modifiedAt` so I don't see much of a downside to adding the `At` suffix.